docs/setup/independent/install-kubeadm.md

```diff
@@ -220,9 +231,44 @@ systemctl enable kubelet && systemctl start kubelet
 
 {% endcapture %}
 
+{% capture coreos %}
+
+Install CNI plugins (required for most pod network):
+
+```bash
+CNI_VERSION="v0.6.0"
+mkdir -p /opt/cni/bin
+curl -L "https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-amd64-${CNI_VERSION}.tgz" | tar -C /opt/cni/bin -xz
+```
+
+Install `kubeadm`, `kubelet`, `kubectl` and add a `kubelet` systemd service:
+
+```bash
+RELEASE="$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)"
```

Comment:
nit: `dl.k8s.io/release/stable.txt` is a better shorthand for this

Reply:
Done